### PR TITLE
Feature/GPS Shift support

### PR DESCRIPTION
**Pull request by dboltovskyi**
ATF Test Scripts to check #[2639](https://github.com/smartdevicelink/sdl_core/issues/2639)

This PR is **ready** for review.

### Summary
Scripts for "Adding GPS Shift support" feature

### ATF version
[develop](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/API/VehicleData/GpsShiftSupport/005_GetInteriorVehicleData_stationLocation_shift.lua [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+runner.Step("Register App", common.registerAppWOPTU)
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+for _, v in pairs(common.shiftValue) do
+    runner.Step("GetInteriorVehicleData RADIO module, shifted " .. tostring(v), getInteriorVehicleData, { v })
+end
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", common.postconditions)
```
Comment (ShobhitAd) [inline test_scripts/API/VehicleData/GpsShiftSupport/005_GetInteriorVehicleData_stationLocation_shift.lua]: Test corrupts the preloaded policy table without the following fix
```suggestion
runner.Step("Stop SDL", commonRC.postconditions)
```

File: test_scripts/API/VehicleData/GpsShiftSupport/006_GetInteriorVehicleData_stationLocation_without_shift.lua [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerAppWOPTU)
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+runner.Step("GetInteriorVehicleData RADIO module, without shifted", common.getInteriorVehicleData, { nil })
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", common.postconditions)
```
Comment (ShobhitAd) [inline test_scripts/API/VehicleData/GpsShiftSupport/006_GetInteriorVehicleData_stationLocation_without_shift.lua]: Test corrupts the preloaded policy table without the following fix
```suggestion
runner.Step("Stop SDL", commonRC.postconditions)
```

File: test_scripts/API/VehicleData/GpsShiftSupport/007_OnInteriorVehicleData_stationLocation_shift.lua [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+runner.Step("Activate App", common.activateApp)
+runner.Step("Subscribe on InteriorVehicleData, RADIO module", common.getInteriorVehicleData, { nil, true })
+
+runner.Title("Test")
+for _, v in pairs(common.shiftValue) do
+    runner.Step("OnInteriorVehicleData, RADIO module, shifted " .. tostring(v), onInteriorVehicleData, { v })
+end
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", common.postconditions)
```
Comment (ShobhitAd) [inline test_scripts/API/VehicleData/GpsShiftSupport/007_OnInteriorVehicleData_stationLocation_shift.lua]: Test corrupts the preloaded policy table without the following fix
```suggestion
runner.Step("Stop SDL", commonRC.postconditions)
```

File: test_scripts/API/VehicleData/GpsShiftSupport/008_OnInteriorVehicleData_stationLocation_without_shift.lua [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerAppWOPTU)
+runner.Step("Activate App", common.activateApp)
+runner.Step("Subscribe on InteriorVehicleData, RADIO module", common.getInteriorVehicleData, { nil, true })
+
+runner.Title("Test")
+runner.Step("OnInteriorVehicleData, RADIO module, without shifted ", common.onInteriorVehicleData, { nil })
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", common.postconditions)
```
Comment (ShobhitAd) [inline test_scripts/API/VehicleData/GpsShiftSupport/008_OnInteriorVehicleData_stationLocation_without_shift.lua]: Test corrupts the preloaded policy table without the following fix
```suggestion
runner.Step("Stop SDL", commonRC.postconditions)
```

File: test_scripts/API/VehicleData/GpsShiftSupport/009_GetInteriorVehicleData_stationLocation_shift_after_subscription.lua [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerAppWOPTU)
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+runner.Step("GetInteriorVehicleData subscription RADIO module, shifted true", getInteriorVehicleData, { true })
+runner.Step("GetInteriorVehicleData RADIO module subscribed, without shifted", getInteriorVehicleDataSubscribed)
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", common.postconditions)
```
Comment (ShobhitAd) [inline test_scripts/API/VehicleData/GpsShiftSupport/009_GetInteriorVehicleData_stationLocation_shift_after_subscription.lua]: Test corrupts the preloaded policy table without the following fix
```suggestion
runner.Step("Stop SDL", commonRC.postconditions)
```

**Comment by TMyza** [comment]
@ShobhitAd All comments have been processed. Please take a look.

**Pull request merged**